lib/activerecord/delay_touching.rb

```diff
@@ -87,17 +87,23 @@ def self.touch_records(attr, klass, records)
           changes[column] = current_time
           records.each do |record|
             next if record.destroyed?
+            next unless record.persisted? # A new record may have been rolled back
```

Comment:
This will only fix new records, how about old records? Is there a way to check if the record was rolled back so we can NOT do the touching?